src/fe-text/irssi.c

```diff
@@ -352,6 +353,7 @@ int main(int argc, char **argv)
 	   you have to call setlocale(LC_ALL, "") */
 	setlocale(LC_ALL, "");
 
+	loglev = g_log_set_always_fatal(G_LOG_FATAL_MASK | G_LOG_LEVEL_CRITICAL);
```

Comment:
g_error is always fatal, so we turn it into g_critical. The commit #191 which @ahf merged and I'm not happy with, will crash your irssi instantly if you accidentally `/reload` a broken config file. By using g_critical, we will be able to inform the user gracefully on `/reload` but still refuse to load this config on initial start by temporarily raising the fatality here. It is re-set on line 365.